include/rtl_433_devices.h

```diff
@@ -113,7 +113,9 @@
 		DECL(m_bus_mode_f) \
 		DECL(wssensor) \
 		DECL(wt1024) \
-		DECL(tpms_pmv107j)
+		DECL(tpms_pmv107j) \
+		DECL(xc0324)
```

Comment:
the file/device should probably be named digitech_xc0324. One more info to find it by.

Reply:
Done :-)